lib/runner/cli/cli.js

```diff
@@ -395,6 +395,12 @@ class CliRunner {
       return false;
     }
 
+    if (this.argv.debug) {
+      Logger.info('Disabling parallelism while running in debug mode');
+
+      return false;
+    }
```

Comment:
We would not need this `this.test_settings.testWorkersEnabled` will return false when debug is true.

Reply:
We should also move the logging to settings.js as currently it the log would never be printed.

Reply:
Nice catch, though I would prefer the log to be in `cli.js` will refactor to incorporate it somehow.

Reply:
updated, @gravityvi can you take a look once more.